src.ts/verkleFFIBindings/getTreeKey.ts

```diff
@@ -1,19 +1,20 @@
-import { Context } from '../wasm/rust_verkle_wasm.js'
+import { Context as VerkleFFI } from '../wasm/rust_verkle_wasm.js'
 
 // This is equal to 2n + 256n * 64n.
 //
 // It is a constant that is used in the `getTreeKeyHashJs` function.
 // See the `getTreeKeyHashJs` function for more details.
 const firstChunk = new Uint8Array([2, 64, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0])
 
+const ffi = new VerkleFFI()
```

Comment:
Can we then expand on this pattern and completely skip the export of `VerkleFFI` in [src.ts/index.ts](https://github.com/ethereumjs/verkle-cryptography-wasm/blob/master/src.ts/index.ts) and instead rename this file here `getTreeKey.ts` -> `verkleFFI.ts` and also add the other methods and then export the methods in plain format?

Then we can generally skip this object wrapping which will likely cause problems generally when it gets necessary to move this instantiated `VerkleFFI()` object around (on the monorepo usage side) otherwise.

(an alternative might be to already do this on the Rust side, not that much of an expert here though).

Reply:
Just implemented those changes.